exporter.go

```diff
@@ -422,3 +431,21 @@ func (e *Exporter) addOrReuseLayer(image imgutil.Image, layer layers.Layer, prev
 	e.Logger.Debugf("Layer '%s' SHA: %s\n", layer.ID, layer.Digest)
 	return layer.Digest, image.AddLayerWithDiffID(layer.TarPath, layer.Digest)
 }
+
+func (e *Exporter) makeBuildReport(layersDir string) (BuildReport, error) {
+	var out []BOMEntry
+	for _, bp := range e.Buildpacks {
+		var bpBuildReport BuildReport
+		bpBuildTOML := filepath.Join(layersDir, bp.ID, "build.toml")
+		if _, err := toml.DecodeFile(bpBuildTOML, &bpBuildReport); os.IsNotExist(err) {
+			// nop
+		} else if err != nil {
+			return BuildReport{}, err
+		}
+		for _, entry := range bpBuildReport.BOM {
+			entry.Buildpack = bp.noAPI().noHomepage()
```

Comment:
should we use `withBuildpack`? https://github.com/buildpacks/lifecycle/pull/461/files#diff-5dce70063c4b1b07e611b3fcd32da0445a0b95970028c8992beb9c955e38320fR361